src/allotropy/parsers/novabio_flex2/novabio_flex2_structure.py

```diff
@@ -93,9 +99,30 @@ def create(series: pd.Series[Any]) -> Sample:
                     if data.get(float, raw_name) is not None
                 ]
             ),
-            properties=properties,
+            cell_counter_properties=cls._get_properties(data, CELL_COUNTER_MAPPINGS),
+            blood_gas_properties=cls._get_properties(
+                data, BLOOD_GAS_DETECTION_MAPPINGS
+            ),
+            osmolality_properties=cls._get_properties(
+                data, OSMOLALITY_DETECTION_MAPPINGS
+            ),
+            ph_properties=cls._get_properties(data, PH_DETECTION_MAPPINGS),
+            cell_type_processing_method=data.get(str, "Cell Type"),
```

Comment:
I know this is not accessed on class properties but ditto any value to this as a constant?

Reply:
Ditto